nss-day-cohort-48/bangazon-tenntom
# User changes based on who is logged in

Change to profile.py top make sure the user changes depending on who is logged in. 

## Changes

An incorrect user profile was returned from Postman with the same user profile returned regardless of the authentication token provided. This fix addresses that issue. 

Made changes to the profile.py file.

## Steps to test:

Git fetch --all
Pull branch tlc-user-same
From the command line, run ./seed_data.sh
start the server python3 manage.py runserver
In Postman, paste this key into the Authorization header. Token 9ba45f09651c5b0c404f37a2d2572c026c146694
From Postman, request profile . http://localhost:8000/profile
confirm that information is returned for user Joe Shepherd with an id of 5. 
In Postman, paste this key into the Authorization header. Token Token 9ba45f09651c5b0c404f37a2d2572c026c146688
From Postman, request profile . http://localhost:8000/profile
confirm that information is returned for user is David Jisie with an id of 6. 

## Related Issues
- Fixes #22
